epochStart/metachain/auctionListSorting.go

```diff
@@ -0,0 +1,104 @@
+package metachain
+
+import (
+	"bytes"
+	"math/big"
+	"sort"
+
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+func (als *auctionListSelector) selectNodes(
```

Comment:
could be extracted also to a new struct: auctionListSorting that does the sorting.

Reply:
I think it's better to leave it as it is now, since I will probably split the code anyway in a future PR which requires to create API data for auction config.

Reply:
ok